src/mui-setup.js

```diff
@@ -16,6 +16,25 @@ export default function muiSetup(sproutBase) {
       display: 'table',
       width: '100%',
       tableLayout: 'fixed',
+      ...sproutBase.overrides.MuiTableRow.root,
+    };
+
+    const cellStyle = {
+      maxHeight: '3.7em',
```

Comment:
why 3.7?

Reply:
basically at least to show two rows in the table